Command/PluginSynchronizeCommand.php

```diff
@@ -41,33 +49,31 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return $isEnabled === true;
         }));
 
-        $this->runCommand([
-            'command' => 'plugin:refresh',
-        ], $output);
-
         foreach ($disabledPlugins as $disabledPlugin) {
             $this->runCommand([
                 'command' => 'plugin:uninstall',
                 'plugins' => [$disabledPlugin],
             ], $output);
         }
 
+        $pluginsActivated = [];
```

Comment:
This variable actually contains a value for every plugin and those with  `1` are failed, so I'll suggest $failedPlugins be used here instead.